grizli/utils.py

```diff
@@ -1371,14 +1384,16 @@ def calc_header_zeropoint(im, ext=0):
         fi = None
 
     # Get AB zeropoint
-    if 'PHOTFNU' in header:
-        ZP = -2.5*np.log10(header['PHOTFNU'])+8.90
-        ZP += 2.5*np.log10(scale_exptime)
-    elif 'PHOTFLAM' in header:
+    if 'PHOTFLAM' in header:
         ZP = (-2.5*np.log10(header['PHOTFLAM']) - 21.10 -
               5*np.log10(header['PHOTPLAM']) + 18.6921)
 
         ZP += 2.5*np.log10(scale_exptime)
+        print(np.log10(header['PHOTFLAM']), np.log10(header['PHOTPLAM']), scale_exptime)
```

Comment:
Should this be logged or is it just debugging?

Reply:
Just debugging. Removed here: b531805f5953180248c849306194ebdda2837e21